image_view/src/image_saver_node.cpp

```diff
@@ -93,7 +93,8 @@ ImageSaverNode::ImageSaverNode(const rclcpp::NodeOptions & options)
       &ImageSaverNode::callbackWithoutCameraInfo, this, std::placeholders::_1),
     hints.getTransport());
 
-  g_format = this->declare_parameter("filename_format", std::string("left%04i.%s"));
+  g_format = this->declare_parameter("filename_format", std::string("frame%04i.%s"));
+  fps_ = this->declare_parameter("fps", 0.1);
```

Comment:
we won't want to change the default behavior of the image saver node - so the default here should probably be 0.0 (in which case we don't do the fps-defined behavior)